apps/e2e/tests/backend/endpoints/api/v1/auth/otp/sign-in.test.ts

```diff
@@ -82,13 +84,49 @@ it("should sign in users created with the server API even if sign up is disabled
   `);
 });
 
+it("should not be able to sign in if email is not verified", async ({ expect }) => {
```

Comment:
as above; the noun is the endpoint `/endpoints/api/v1/auth/otp/sign-in`

```suggestion
it("should not allow signing in if email is not verified", async ({ expect }) => {
```